sequencing/system.py

```diff
@@ -360,6 +360,56 @@ def set_cross_kerr(self, mode1, mode2, chi=0):
         )
         self.cross_kerrs[key] = chi
 
+    def set_sixth_order_coupling(self, mode1, mode2, chip=0):
```

Comment:
Ugh this should be removed; updates are in this PR: https://github.com/qci-chen/sequencing/pull/1

Reply:
This comment that the updates are in #1 is not longer true, not sure why...

Reply:
maybe the comment meant #4 or #6 ?